urls.py

```diff
@@ -940,4 +950,126 @@
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/moving-pictures/emp-single-end-sequences/sequences.fastq.gz',
+    '2017.11/tutorials/metadata/faith_pd_vector.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/metadata/faith_pd_vector.qza',
+    '2017.11/tutorials/metadata/rep-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/metadata/rep-seqs.qza',
+    '2017.11/tutorials/metadata/taxonomy.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/metadata/taxonomy.qza',
+    '2017.11/tutorials/metadata/unweighted_unifrac_pcoa_results.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/metadata/unweighted_unifrac_pcoa_results.qza',
+    '2017.11/tutorials/quality-control/query-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/quality-control/query-seqs.qza',
+    '2017.11/tutorials/quality-control/reference-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/quality-control/reference-seqs.qza',
+    '2017.11/tutorials/quality-control/query-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/quality-control/query-table.qza',
+    '2017.11/tutorials/sample-classifier/atacama-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/sample-classifier/atacama-table.qza',
+    '2017.11/tutorials/sample-classifier/moving-pictures-table.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/sample-classifier/moving-pictures-table.qza',
+    '2017.11/tutorials/training-feature-classifiers/85_otu_taxonomy.txt':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/training-feature-classifiers/85_otu_taxonomy.txt',
+    '2017.11/tutorials/training-feature-classifiers/85_otus.fasta':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/training-feature-classifiers/85_otus.fasta',
+    '2017.11/tutorials/training-feature-classifiers/rep-seqs.qza':
+        'https://s3-us-west-2.amazonaws.com/qiime2-data/2017.11/tutorials/training-feature-classifiers/rep-seqs.qza',
+    '2017.11/tutorials/longitudinal/sample_metadata.tsv':
+        'https://docs.google.com/spreadsheets/d/e/2PACX-1vQBO3snrJj1tBS3xhegwTFODYUgr5kct66npoXPxRyQ8CZXvxcszrXRVx-NXR-aJehapb1a6k6lZQNr/pub?gid=1303657428&single=true&output=tsv',
+    '2017.11/tutorials/longitudinal/sample_metadata':
+        'https://docs.google.com/spreadsheets/d/1B_mPluLmKWuDxZmH-gkpO1FnWfOGDP8JSUhpxBCJ0Gg/edit?usp=sharing',
+    '2017.11/tutorials/atacama-soils/sample_metadata.tsv':
+        'https://docs.google.com/spreadsheets/d/13jmPk_QByAFZV5TRKYERdgf3OwuFbv90tbLLoCJSdFg/export?gid=0&format=tsv',
+    '2017.11/tutorials/atacama-soils/sample_metadata':
+        'https://docs.google.com/spreadsheets/d/13jmPk_QByAFZV5TRKYERdgf3OwuFbv90tbLLoCJSdFg/edit?usp=sharing'
```

Comment:
travis says there's a missing comma here ⛔️

Reply:
Fixed --- not sure why flake8 didn't catch this locally...

Reply:
Oh, looks like my `vim` plugins are messed up on my new dev machine! Thanks for the alert!